src/test/java/uk/gov/justice/hmpps/prison/repository/ReferenceDataRepositoryTest.java

```diff
@@ -77,7 +104,7 @@ public void testGetReferenceCodesByDomainWithoutSubCodes1() {
         assertThat(refCodes.getItems()).isNotNull();
         assertThat(refCodes.getItems()).isNotEmpty();
         assertThat(refCodes.getItems()).hasSize(10);
-        assertThat(refCodes.getTotalRecords()).isEqualTo(58);
+        assertThat(refCodes.getTotalRecords()).isEqualTo(59);
```

Comment:
not your fault, but this is all really naff.  We shouldn't have to fix lots of tests when we want to add in more test reference data :angry:

Reply:
I was thinking the same. I could assert the result with the json form saved to file, would that be better? or verify results are sorted as expected with any result set so that no code change is required when new records are inserted but would fail if the sorting is incorrect. Would that be better?

Reply:
this style is used in other places, for financial transactions for example, and related tables, on a previous ticket I inserted a few new rows and it impacted many tests (counts and totals, etc) so it makes inserting new custom records harder and more time-consuming.